### DISCUSSION TITLE
1.8.3

### DISCUSSION THREAD

**Discussion by yujincheng08** [LSPosed/LSPosed#1924] [New Release]
### Changelog
- Fix module not working normally on Android 10
- Refine notification on module installation (thanks @fankes)
- Fix hook may not work on debuggable apps/modules
- Fix XSharedPreference after a soft reboot
- Support Android 13 beta 1
- Fix zygisk installation on x86 only devices
- Fix manager state restoration

### 更新日志
- 修复 Android 10 下模块工作异常
- 完善模块安装通知（感谢 @fankes）
- 修复调试模式下模块或应用钩子可能失效
- 修复软重启后的 XSharedPreference
- 支持 Android 13 beta 1
- 修复仅 x86 设备下 zygisk 版本的安装
- 修复管理器状态恢复

<hr /><em>This discussion was created from the release <a href='https://github.com/LSPosed/LSPosed/releases/tag/v1.8.3'>1.8.3</a>.</em>

**Comment by 2053628642** [comment]
为什么设置模块界面会显示我的名字和XSpace

**Reply by david082321** [reply]
这是你的手机厂商设置的名称。

LSPosed 只是如实显示你手机提供的数据。

**Reply by node93** [reply]
pm compile gives error in lsposed because of dex optimiser wrapper even though it says supported please provide a toggle to turn off this option I'm using lsposed 1.8.0 because of this. Using on A11.

**Reply by MahuuhaM** [reply]
有人会安装吗，我搞不懂

**Comment by node93** [comment]
dex warpper is destroying dalivik cache

**Reply by chuppito** [reply]
the bluetooth stops working with the latest version.  I tried to put version 1.8.2 but lsposed tells me that it is not activated.
 I have to switch between bluetooth and lsposed.
 I'm on Android 13

**Comment by Peterepeat87** [comment]
If anyone else is having the issue whereby LSPosed won't open / crashes when opened, ensure that Shell is not included in the Zygisk Deny List. 
Seems silly, but it gave me a headache for some time.
